style.css

```diff
@@ -630,3 +647,430 @@
+    font-family: var(--font-family);
+    color: #091e42;
+    padding: 5px;
+    margin-top: -0.5vw;
+    margin-left: 0.8rem;
+  }
+
+  .desktop-work-sec .div-info .ul-li {
+    display: flex;
+    justify-content: space-around;
+    font-family: var(--font-family);
+    font-size: 18px;
+    color: #6b778c;
+    margin-right: 9vw;
+    margin-left: -2.6vw;
+    position: relative;
+    margin-top: -2vw;
+  }
+
+  .desktop-work-sec .div-info .sec-list {
+    font-family: var(--font-family);
+    font-size: 12px;
+    color: #6070ff;
+    position: relative;
+    margin-left: -2.1rem;
+    margin-top: -1vw;
+  }
+
+  .div-info button {
+    font-family: var(--font-family);
+    font-size: 17px;
+    color: #396df2;
+    background-color: #fff;
+    border: 1px solid #396df2;
+    border-radius: 8px;
+    padding: 0.5rem 1rem;
+    width: 9rem;
+    height: 3.5rem;
+    margin-left: 1rem;
+    margin-top: 0.5vw;
+    text-align: center;
+  }
+
+  .flex {
+    display: grid;
+    grid-template-columns: 30vw 50vw;
+    gap: 1rem;
+    background-color: #fff;
+    margin-top: 145rem;
+    padding: 0 6vw 6rem 6vw;
+    border-top-right-radius: 100px;
+  }
+
+  .flex .flex-box {
+    grid-column: 1;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .flex .flex-box .social-img {
+    position: relative;
+    margin-left: -3rem;
+  }
+
+  button:hover {
+    cursor: pointer;
+    background-color: #6070ff;
+    color: #fff;
+  }
+
+  .contact-form form button {
+    font-family: var(--font-family);
+    font-size: 17px;
+    color: #396df2;
+    background-color: #fff;
+    border: 1px solid #396df2;
+    border-radius: 8px;
+    padding: 0.5rem 1rem;
+    width: 9rem;
+    height: 3rem;
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    margin: 0.5vw 23vw 0 23vw;
+  }
+
+  .flex .flex-box button {
+    font-family: var(--font-family);
+    font-size: 17px;
+    color: #396df2;
+    background-color: transparent;
+    border: 1px solid #396df2;
+    border-radius: 8px;
+    padding: 0.5rem 1rem;
+    width: 9rem;
+    height: 3rem;
+    margin-left: 0;
+    margin-top: 0.5vw;
+    text-align: center;
+  }
+
+  .flex-box-2 .flex-3 {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .flex-3 .grid .more-img {
+    float: right;
+    margin-top: -4rem;
+  }
+
+  .flex-3 .grid .grid-2 .language {
+    position: relative;
+    margin-left: -3rem;
+  }
+
+  .flex-3 .li-2 img {
+    float: right;
+    margin-top: -4rem;
+  }
+
+  .flex-3 .li-3 img {
+    float: right;
+    margin-top: -4rem;
+  }
+
+  .flex-3 .line {
+    height: 1px;
+    background-color: #dfe1e6;
+    width: 50vw;
+  }
+
+  .contact-form {
+    background-image: url("images/contact\ form\ background\ shapes@2x.png");
+    background-repeat: no-repeat;
+    background-size: 100%;
+    background-color: #6070ff;
+    border-top-left-radius: 100px;
+    height: 800px;
+    padding: 3rem 20vw 0 20vw;
+  }
```

Comment:
- [ ] Please 🙏  make sure that your description paragraph (`<p>` tag) of `contact-form` looks just like that in Figma design and make your text align center instead of Justify.  I also recommend adjusting `font-weight` accordingly for the para, as mentioned in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#project-requirements): 

> You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).


**_Figma Design_**

<img width="368" alt="Screenshot 2023-02-23 at 9 11 50 PM" src="https://user-images.githubusercontent.com/44798044/220983700-3842f9ff-bf6f-4b6b-90da-58b9df714086.png">

_**Your design**_

<img width="511" alt="Screenshot 2023-02-23 at 9 12 03 PM" src="https://user-images.githubusercontent.com/44798044/220983606-6d3fc068-e389-4956-92fb-09417eb6a506.png">